app/navigation/Navigator.scala

```diff
@@ -197,6 +197,7 @@ class Navigator @Inject() () {
     case p: EmailPage                              => _ => routes.CyaRequestAdviceController.onPageLoad(p.recordId)
     case p: AssessmentPage                         => navigateFromAssessmentCheck(p)
     case p: HasSupplementaryUnitPage               => navigateFromHasSupplementaryUnitCheck(p.recordId)
+    case p: HasCountryOfOriginChangePage           => answers => navigateFromHasCountryOfOriginChangeCheck(answers, p.recordId)
```

Comment:
Missing unit test for this

Reply:
This should also be in the normal mode section not the check mode